source/PowerUp.js

```diff
@@ -108,7 +386,7 @@ export class RapidFire extends PowerUp {
           game.addChild(new Projectile(
               curShip.position.x + curShip.width,
               curShip.position.y + curShip.height/2,
-              curShip, this.fireRate, this.bulletSpeed, this.projectileType))
+              this.vector, curShip, this.bulletSpeed, this.projectileType))
       }
```

Comment:
The BFG is supposed to have kickback equal to the ship's normal forward velocity, so you'll need to add something like `curShip.move(-curShip.speed * delta, "x")`